litestar/router.py

```diff
@@ -259,17 +260,15 @@ def register(self, value: ControllerRouterHandler) -> list[BaseRoute]:
 
     @property
     def route_handler_method_map(self) -> dict[str, RouteHandlerMapItem]:
-        """Map route paths to :class:`RouteHandlerMapItem <litestar.types.internal_typ es.RouteHandlerMapItem>`
+        """Map route paths to :class:`RouteHandlerMapItem <litestar.types.internal_types.RouteHandlerMapItem>`
```

Comment:
```suggestion
        """Map route paths to :class:`~litestar.types.internal_types.RouteHandlerMapItem`
```
I've noticed @JacobCoffee (aka docs-god) doing this lately, it only renders in the docs as `RouterHandlerMapItem`.